src/main/java/nl/tudelft/contextproject/script/UpdateTask.java

```diff
@@ -0,0 +1,20 @@
+package nl.tudelft.contextproject.script;
+
+import nl.tudelft.contextproject.ContextTFP;
+
+import java.util.TimerTask;
+
+public class UpdateTask extends TimerTask {
```

Comment:
About this class, seeing as you're only calling its constructor from Script.updateOldCamCaller(), I think it would be better to replace `new UpdateTask()` there with a lambda expression. Using an inner class like you had a few commits ago was fine by me too, but not as a public inner class, rather a static inner class (`static class UpdateTask`). Lambda expressions are the best though